test/e2e/backup_restore.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Add readme to describe how to run the tests and if any environment variables need to be set up if running in dev mode

Reply:
Done :+1: